src/bpftrace.cpp

```diff
@@ -724,8 +726,86 @@ void perf_event_lost(void *cb_cookie, uint64_t lost)
   bpftrace->out_->lost_events(lost);
 }
 
-std::unique_ptr<AttachedProbe> BPFtrace::attach_probe(Probe &probe, const BpfOrc &bpforc)
+std::vector<std::unique_ptr<AttachedProbe>> BPFtrace::attach_usdt_probe(
+    Probe &probe,
+    std::tuple<uint8_t *, uintptr_t> func,
+    int pid,
+    bool file_activation)
+{
+  std::vector<std::unique_ptr<AttachedProbe>> ret;
+
+  // File activation works by scanning through /proc/*/maps and seeing
+  // which processes have the target executable in their address space
+  // with execute permission. If found, we will try to attach to each
+  // process we find.
+  if (file_activation && probe.path.size())
```

Comment:
```suggestion
  if (!(file_activation && probe.path.size()))
  {
       ret.emplace_back(std::make_unique<AttachedProbe>(probe, func, pid));
       return ret;
   }
```

Avoiding the big if statement might improve readability a bit, nit.